rules/Naming/Rector/Class_/RenamePropertyToMatchTypeRector.php

```diff
@@ -149,6 +150,6 @@ private function skipDateTimeOrMockObjectPropertyType(Property $property): bool
             return true;
         }
 
-        return $this->isName($property->type, ClassName::DATE_TIME);
+        return $this->isObjectType($property->type, new ObjectType(ClassName::DATE_TIME_INTERFACE));
```

Comment:
Could you add isObjectType on `ClassName::MOCK_OBJECT` as well above on line 149? Thank you.